src/components/Toast/ToastLink/ToastLink.module.scss

```diff
@@ -3,7 +3,7 @@
 
 .actionLink {
   @include theme-prop(color, text-color-on-primary);
-  text-decoration: underline;
+  text-decoration: underline !important;
```

Comment:
that should probably be a prop for the `TextLink` component

Reply:
what do you mean? Right now we don't have link component only link skin. 
The reason we want to add here the underline is because  link in on primary is in the same color as text... maybe we can just set underline as default on link skin with onPrimary color?

Reply:
As I see it - the toast link button should be either the `<Link/>` which can take the appropriate skin (onPrimary?) and have the ability to control the underline (e.g. never, onHover, always)

Reply:
After I discuss with Meirav I added the link appearance behaviour to the component behviour (no underline unless it's primary, that's the appearance and behivour Meirav want).